src/caf/space/ui.py

```diff
@@ -798,38 +797,38 @@ def _redirect_logging(self):
         self.console_handler.setFormatter(fmt)
         self.console_handler.setLevel(logging.INFO)
         self._logger.addHandler(self.console_handler)
-        self._logger.info("Log file saved here: %s", (Path(os.getcwd()) / SpaceUI._LOG_NAME))
+        # self._logger.info("Log file saved here: %s", (Path(os.getcwd()) / SpaceUI._LOG_NAME))
 
 
 class SpaceUI:
     """
     Main class to launch UI.
     """
 
-    _LOG_NAME = "SPACE.log"
+    # _LOG_NAME = "SPACE.log"
 
     def __init__(self):
-        self.log_file = Path(os.getcwd()) / self._LOG_NAME
+        # self.log_file = Path(os.getcwd()) / self._LOG_NAME
         # Remove log file if present
-        if os.path.exists(self.log_file):
-            os.remove(self.log_file)
+        # if os.path.exists(self.log_file):
+        #     os.remove(self.log_file)
 
         # Initiate logger object
         self.logger = logging.getLogger("SPACE")
         self.logger.setLevel(logging.DEBUG)
```

Comment:
Get the `__package__` logger to be consistent

```suggestion
        self.logger = logging.getLogger(__package__)
        self.logger.setLevel(logging.DEBUG)
```